contracts/registry/src/lib.rs

```diff
@@ -885,4 +882,47 @@ mod tests {
         );
         assert_eq!(test_utils::get_logs(), vec![log_mint, log_renew].concat());
     }
+    #[test]
+    fn sbt_recover_basics() {
+        let (_, mut ctr) = setup(&issuer1(), 3 * MINT_DEPOSIT);
+        let m1_1 = mk_metadata(1, Some(START + 10));
+        let m2_1 = mk_metadata(2, Some(START + 10));
+        ctr.sbt_mint(vec![(alice(), vec![m1_1.clone(), m2_1.clone()])]);
```

Comment:
could you mint a new token to `alice` using different issuer, and later in the test verify that alice will still hold that token?